docs/multicluster/user-guide.md

```diff
@@ -399,13 +399,11 @@ supported.
 After the Gateway is created, Multi-cluster Controller will be responsible
 for exporting the cluster's network information to other member clusters
 through the leader cluster, including the cluster's Gateway IP and Service
-CIDR. Multi-cluster Controller will try to discover the cluster's Service CIDR
-automatically, but you can also manually specify the `serviceCIDR` option in
-ConfigMap `antrea-mc-controller-config`. In other member clusters, a
-ClusterInfoImport CR will be created for the cluster which includes the
-exported network information. For example, in cluster `test-cluster-west`, you
-you can see a ClusterInfoImport CR with name `test-cluster-east-clusterinfo`
-is created for cluster `test-cluster-east`:
+CIDR. Please check [Member Cluster Service CIDR Discovery](#member-cluster-service-cidr-discovery).
```

Comment:
How about we put this sentence in () after "Service CIDR", like:

`including the cluster's Gateway IP and Service (check [Member Cluster Service CIDR Discovery](#member-cluster-service-cidr-discovery)).`